css/fonts.css

```diff
@@ -0,0 +1,36 @@
+@font-face {
```

Comment:
This file seems ok, too... did you write it or is it from somewhere else? Then, we should include the license.

Reply:
https://google.github.io/material-design-icons/#icon-font-for-the-web

Reply:
https://github.com/google/material-design-icons

Reply:
Ok. Could you put that link as a comment into the first lines of the file?

Reply:
It does not require a license, I hope, because it is so small.